source/Octopus.Server.Client/Model/SpaceLicenseUsageResource.cs

```diff
@@ -0,0 +1,9 @@
+namespace Octopus.Client.Model;
+
+public class SpaceLicenseUsageResource
+{
+    public string SpaceName { get; set; } = string.Empty;
+    public int Projects { get; set; }
```

Comment:
Nit: ProjectsCount might make more sense here. In the context of the SpaceLicenseUsageResource class, the Projects/Tenants/Machines naming makes sense so no need to actually change this unless you want to.